sample/src/extension.ts

```diff
@@ -37,7 +38,7 @@ export function activate(context: vscode.ExtensionContext) {
             await vscode.commands.executeCommand('dotnet.showAcquisitionLog');
 
             // Console app requires .NET Core 2.2.0
-            const commandRes = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.acquire', { version: '2.2' });
+            const commandRes = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet.acquire', { version: '2.2', requestingExtensionId });
```

Comment:
is this a public API change? We need an additional requestingExtensionId ? What if the user does not provide it?

Reply:
Yes, this is an API change for the extension calling us. If the extension calling acquire does not provide its own ID everything still works except they get this warning: https://github.com/dotnet/vscode-dotnet-runtime/pull/80/files#diff-4ac984a01117704ea53847e086aca20cR57 (cc @StephenWeatherford, as you'll need to accommodate this change once this version ships)

Reply:
Make sure you get an acknowledgement

Reply:
ETA on the release?  Thx.

Reply:
Actually, we're shipping in about a week.  I'm going to start passing it in then.

Reply:
Not sure when the next release will be, we don't have a definite schedule. I would guess in a few weeks.